### ISSUE TITLE
Chaise pages don't render through IE11 accessed via a Terminal server

### ISSUE THREAD

**Issue by robes** [informatics-isi-edu/chaise#1392]
This happened with this setup:
- Mac client
- connecting via a Windows Terminal Server
- IE 11

![image](https://user-images.githubusercontent.com/4202842/33451725-c282b3b0-d5c4-11e7-862a-b09d0de746b9.png)

Somewhat related to #1391 

We might be able to reproduce this by running a Mac and using Remote Desktop to connect to a Windows client and using IE 11 on that remote Windows desktop.

**Comment by hongsudt** [comment]
THis is such an edge case... Why do they have to go through a round-about way to get to the chaise app?

**Comment by karlcz** [comment]
@hongsudt  This is one of the funny deployment modes people use for "secure" infrastructure. They treat the terminal server like a category of DMZ in their network/firewall architecture.

**Comment by hongsudt** [comment]
@robes can you ask your user whether they have this issue with chrome or firefox?  Just want to check whether they have alternative approach that works.

**Comment by robes** [comment]
The user does report that the remote desktop had FF installed and it worked. It did not have Chrome installed.

**Comment by robes** [comment]
@hongsudt I just did a test. I have a Windows 7 Pro host with IE 11. I connected via Remote Desktop to it (from my Mac). Over the RDP, I used IE 11 to browse the www.facebase.org deployment of chaise. It all worked for me. Of course, this does not exactly duplicate the user environment which was a windows terminal server. My vote here would be to leave this issue open but unassigned just as a record of a known issue.
